src/test/mockFactory.ts

```diff
@@ -138,12 +138,37 @@ export class MockFactory {
 
     // TODO: Reduce usage on tokenCache._entries[0]
     credentials["tokenCache"] = {
-      _entries: [{ accessToken: "ABC123" }]
+      _entries: [this.createTestTokenCacheEntries()]
```

Comment:
I don't believe we're using this intern `_entries` member anymore.  Can we refactor this?